### DISCUSSION TITLE
--verbose not working on npx tailwindcss

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#4739] [Help]
@simonswiss I tried your command from your latest video and the below works.

```text
npx tailwindcss -o ./assets/css/tailwind.css --watch
```

When I try to use `--verbose` it does not work.

```text
npx tailwindcss -o ./assets/css/tailwind.css --watch --verbose
```

Is it not supported or am I doing it wrong?

**Comment by adamwathan** [comment]
Yeah there is no `--verbose` option — what would you expect to see there if we included one?

**Reply by jenstornell** [reply]
When comparing the postcss `--verbose` with the tailwind version I now see that they are **not** that different. I still have some suggestions.

## For verbose

- **Filepath:** To know where the file is put, the postcss version output the filepath.
- **Datetime:** Sometimes it's hard to know, did it run or not? To know for sure it would be helpful with a datetime. It's not present in any of the versions.

## For both

- **Colors:** I really like the colors in the postcss version. I think it improves readability a bit (green as a success color etc).
- **Done:** `414ms.` vs `414 ms`  I think the second of these two is the most readable which is the postcss version. It's super duper nitpicky I know.

Else I like that the tailwind output version is shorter. You use `Done` instead of `Finished` which save some space.

![compare](https://user-images.githubusercontent.com/927011/122872632-02df9f00-d331-11eb-96bf-37105e27285a.png)
